### PR TITLE
Update VReplication Reference Pages

### LINKED ISSUE #1242: [Docs Review] VReplication Reference Pages
Parent Issue vitessio/website#1222 
Vitess github url: https://github.com/vitessio/website

### Pages to review:

- [x] [VReplication](https://vitess.io/docs/15.0/reference/vreplication/)
- [x] [Overview](https://vitess.io/docs/16.0/reference/vreplication/vreplication/)
- [x] [MoveTables](https://vitess.io/docs/16.0/reference/vreplication/movetables/)
- [x] [Reshard](https://vitess.io/docs/16.0/reference/vreplication/reshard/)
- [x] ~~[--- Create](https://vitess.io/docs/16.0/reference/vreplication/create/)~~
- [x] ~~[--- SwitchTraffic](https://vitess.io/docs/16.0/reference/vreplication/switchtraffic/)~~
- [x] ~~[--- ReverseTraffic](https://vitess.io/docs/16.0/reference/vreplication/reversetraffic/)~~
- [x] ~~[--- Complete](https://vitess.io/docs/16.0/reference/vreplication/complete/)~~
- [x] ~~[--- Show](https://vitess.io/docs/16.0/reference/vreplication/show/)~~
- [x] ~~[--- Progress](https://vitess.io/docs/16.0/reference/vreplication/progress/)~~
- [x] ~~[--- Cancel](https://vitess.io/docs/16.0/reference/vreplication/cancel/)~~
- [x] [Materialize](https://vitess.io/docs/16.0/reference/vreplication/materialize/)
- [x] [VDiff](https://vitess.io/docs/16.0/reference/vreplication/vdiff/)
- [x] [VDiff2](https://vitess.io/docs/16.0/reference/vreplication/vdiff2/)
- [x] [Workflow](https://vitess.io/docs/16.0/reference/vreplication/workflow/)
- [x] [VExec](https://vitess.io/docs/16.0/reference/vreplication/vexec/)
- [x] [VReplicationExec](https://vitess.io/docs/16.0/reference/vreplication/vreplicationexec/)
- [x] [VTTablet Flags](https://vitess.io/docs/16.0/reference/vreplication/flags/)
- [x] [Metrics](https://vitess.io/docs/16.0/reference/vreplication/metrics/)
- [x] [Migrate](https://vitess.io/docs/16.0/reference/vreplication/migrate/)
- [x] [Mount](https://vitess.io/docs/16.0/reference/vreplication/mount/)
- [x] [Shard Level Migrations](https://vitess.io/docs/16.0/reference/vreplication/shardlevelmigrations/)
- [x] [Tablet selection](https://vitess.io/docs/16.0/reference/vreplication/tablet_selection/)
- [x] [Throttling](https://vitess.io/docs/16.0/reference/vreplication/throttling/)
- [x] [VReplication FAQ](https://vitess.io/docs/16.0/reference/vreplication/faq/)
- [x] ~~[VReplication V1 Commands](https://vitess.io/docs/16.0/reference/vreplication/v1/)~~
- [x] ~~[MoveTables](https://vitess.io/docs/16.0/reference/vreplication/v1/movetables/)~~
- [x] ~~[SwitchReads](https://vitess.io/docs/16.0/reference/vreplication/v1/switchreads/)~~
- [x] ~~[SwitchWrites](https://vitess.io/docs/16.0/reference/vreplication/v1/switchwrites/)~~
- [x] ~~[DropSources](https://vitess.io/docs/16.0/reference/vreplication/v1/dropsources/)~~
- [x] ~~[Reshard](https://vitess.io/docs/16.0/reference/vreplication/v1/reshard/)~~

### What to do 

- [ ] Make sure the content reflect the current state and facts are up-to-date.
- [ ] Add details if needed either in form of new section on current page or adding new page.
- [ ] Add links/Urls on page are working
- [ ] Deprecate flags should be marked as 'deprecated'
- [ ] Review is done for ~~both V15 and~~ V16

Please note that we are not going to focus of restructuring the website content as part of this project. It is something we will do as a separate project later on.

### PR DESCRIPTION
**Pull request by mattlord**
Documents: https://github.com/vitessio/vitess/pull/11705 (likely some stragglers to be corrected in follow-ups as we review the full set of docs)

Fixes: https://github.com/vitessio/website/issues/1242

### CODE REVIEW

File: content/en/docs/16.0/user-guides/migration/cutover.md [hunk 1]
```diff
@@ -168,9 +65,9 @@ Depending on the size of the table(s), the VDiff process may need increased time
 - Increase the `--filtered_replication_wait_time` parameter for VDiff as the default is 30 seconds. You many need to increase this to hours on large and/or busy tables.
 
 {{< info >}}
-Note that running VDiff via vtctld can lead to vtctld consuming significantly more memory than usual. 
-We've found this to be around 1 GB plus, instead of the a few hundred MB that it normally uses. 
-If you run your vtctld in a memory limited container, you may want to take this into account. 
+Note that running VDiff via vtctld can lead to vtctld consuming significantly more memory than usual.
```
Comment (mattlord) [inline content/en/docs/16.0/user-guides/migration/cutover.md:68]: Copying this relevant comment over: https://github.com/vitessio/website/pull/1187/files#r1002133833

File: content/en/docs/16.0/reference/vreplication/metrics.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ description: Metrics related to vreplication functionality
 weight: 85
 ---
 
-VReplication exports several metrics using the expvars interface. These are available at the `debug/vars` endpoint of vttablet's http status pages. [More details here](../../features/monitoring/#3-push-based-metrics-system#3-push-based-metrics-system)
+VReplication exports several metrics using the expvars interface. These are available at the `/debug/vars` endpoint of vttablet's http status pages. [More details can be found here](../../features/monitoring/#3-push-based-metrics-system#3-push-based-metrics-system).
```
Comment (rohit-nayak-ps) [inline content/en/docs/16.0/reference/vreplication/metrics.md]: The anchor is repeated and hence link doesn't work as expected.

File: content/en/docs/16.0/reference/vreplication/movetables.md [hunk 3]
```diff
@@ -246,48 +236,176 @@ We caution against against using `EXEC` or `EXEC_IGNORE` for the following reaso
...
+
+If set, the workflow will stop once the Copy phase has been completed i.e. once
+all tables have been copied and VReplication decides that the lag
+is small enough to start replicating, the workflow state will be set to Stopped.
+
+</div>
+
+###### Uses
+* If you just want a consistent snapshot of all the tables you can set this flag. The workflow
+will stop once the copy is done and you can then mark the workflow as `Complete`d
```
Comment (rohit-nayak-ps) [inline content/en/docs/16.0/reference/vreplication/movetables.md]: `Complete`d doesn't render well, probably best to remove the trailingd.

File: content/en/docs/16.0/reference/vreplication/movetables.md [hunk 4]
```diff
@@ -246,48 +236,176 @@ We caution against against using `EXEC` or `EXEC_IGNORE` for the following reaso
...
+**default** `--vreplication_tablet_type` parameter value for the tablet. `--vreplication_tablet_type` has the default value of "in_order:REPLICA,PRIMARY".\
+**string**
+
+<div class="cmd">
+
+Source tablet types to replicate from (e.g. PRIMARY, REPLICA, RDONLY).
+
+</div>
+
+###### Uses
```
Comment (rohit-nayak-ps) [inline content/en/docs/16.0/reference/vreplication/movetables.md]: Incorrect cut/paste: this is the same text as for `auto_start`

File: content/en/docs/16.0/reference/vreplication/reshard.md [hunk 5]
```diff
@@ -198,28 +226,111 @@ We caution against against using `EXEC` or `EXEC_IGNORE` for the following reaso
...
+**string**
+
+<div class="cmd">
+
+Source tablet types to replicate from (e.g. PRIMARY, REPLICA, RDONLY).
+
+</div>
+
+###### Uses
+* allows updating the rows in `_vt.vreplication` after `Reshard` has setup the
```
Comment (rohit-nayak-ps) [inline content/en/docs/16.0/reference/vreplication/reshard.md]: Same issue here as in MoveTables. Need to change this text.

**Review by rohit-nayak-ps — CHANGES_REQUESTED** [review]
Nice! 

Noticed a few minor issues. Otherwise looks very good.

**Comment by mattlord** [comment]
Thanks, @rohit-nayak-ps ! I believe that I addressed your comments here: https://github.com/vitessio/website/pull/1267/commits/e02d88c52b943f2f15e87fb90e8516958c1baa1e

**Review by harshit-gangal — APPROVED** [review]

File: content/en/docs/16.0/reference/vreplication/tablet_selection.md [hunk 6]
```diff
@@ -1,24 +1,28 @@
 ---
-title: Tablet selection
+title: Tablet Selection
 weight: 300
 ---
 
 ### Introduction
 
-For [VReplication streams](../../../concepts/vstream/), we must choose a tablet to serve the role of source (vstreamer) and target (vapplier) in the replication stream and this is done automatically.
+For both [VTGate VStreams and VTTablet VReplication streams](../../../concepts/vstream/) we must choose a tablet to serve the role of source (vstreamer). For VReplication streams we must also choose a a tablet to serve the role of target (vapplier). This tablet selection is performed by the internal `TabletPicker` component.
```
Comment (rohit-nayak-ps) [inline content/en/docs/16.0/reference/vreplication/tablet_selection.md]: We don't choose a target tablet: vreplication streams are running on the target shard's primary.

File: content/en/docs/16.0/reference/vreplication/tablet_selection.md [hunk 7]
```diff
@@ -1,24 +1,28 @@
...
+The `TabletPicker` will only look for source and target tablet tablets within the same cell of the calling process by default — the `vtgate` managing the VStream or the target `vttablet` for a VReplication stream — so e.g. if the target primary tablet is in the `zone1` cell it will only look for source tablets in the `zone1` cell. If you want to have cross-cell streams then you will need to specify the list of cells or any [CellAlias](https://vitess.io/docs/reference/programs/vtctl/cell-aliases/) that contain the list of potential cells using the `--cells` flag in your VReplication workflow commands like [`MoveTables`](../movetables/) or the [`VStreamFlags.Cell`](https://pkg.go.dev/vitess.io/vitess/go/vt/proto/vtgate#VStreamFlags) field in a [`VStreamRequest`](https://pkg.go.dev/vitess.io/vitess/go/vt/proto/vtgate#VStreamRequest).
```
Comment (rohit-nayak-ps) [inline content/en/docs/16.0/reference/vreplication/tablet_selection.md]: typo: "tablet tablets"

**Review by rohit-nayak-ps — CHANGES_REQUESTED** [review]

File: content/en/docs/16.0/reference/vreplication/tablet_selection.md (hunk 6 above)
Comment (mattlord) [reply content/en/docs/16.0/reference/vreplication/tablet_selection.md]: Good catch! Mistake in modifying that paragraph. I've cleaned it up here: https://github.com/vitessio/website/pull/1267/commits/e4696ec3fbe3ac4ace100d4ce919331cc0535af3

**Review by rohit-nayak-ps — APPROVED** [review]

**Pull request merged**